src/NServiceBus.AcceptanceTests/Recoverability/When_immediate_retries_fail.cs

```diff
@@ -23,8 +23,7 @@ public class When_immediate_retries_fail : NServiceBusAcceptanceTest
 
                 .Run();
 
-            Assert.GreaterOrEqual(1, context.NumberOfDelayedRetriesPerformed, "Should only do one retry");
-            Assert.GreaterOrEqual(context.TimeOfSecondAttempt - context.TimeOfFirstAttempt, Delay, "Should delay the retry");
+            Assert.GreaterOrEqual(2, context.NumberOfTimesInvoked, "Should only do one retry");
```

Comment:
"Should only do one retry" seems to no longer be correct message here, since this will only fail if <= 1, but will pass if 2 or more.